### DISCUSSION TITLE
Is it possible to adapt _TW to use a structure similar to Twenty Twenty-Three?

### DISCUSSION THREAD

**Discussion by CGTS** [gregsullivan/_tw#36] [Q&A] [answered]
I've used _TW in the past and liked how quickly I could get things set up and start coding. However when creating blocks I would use have [LazyBlocks](https://www.lazyblocks.com/) to build them out. With the new structure it looks like its much easier to create in the parts/patterns/templates folder and the html markup. 

I'd like to use the create block theme plugin to create a blank theme and integrate tailwind similar to how _TW works. Is there any reason this wouldn't work?

**Comment by gregsullivan** [comment, marked as answer]
I think for the most part it should work!

Global styles were the one element of block themes and site editing in general that I could never find a way to make work with Tailwind. I can't remember specifics, but I think there were issues with specificity that conflicted with Tailwind in a way I was never able to resolve.

I'm not yet at the point where I'm using block themes for client work, so I haven't taken a deep dive on this one. I'll likely expand _tw's support for site editing as a hybrid theme before I start using a true block theme. (And depending on how site editing functionality is ported back to hybrid themes, it's possible that for many site builds a hybrid theme will continue to make sense, at least for me personally.)

All of that said, I'm fairly confident that both the `patterns` and `templates` folders would work with _tw as well. (Note that block templates are disabled by default in `functions.php`.)

Hope that helps!

**Reply by CGTS** [reply]
Thanks for reply. I'll dig into it and see what happens.

**Reply by gregsullivan** [reply]
No problem—please let me know how it goes!
